components/camera/platforms/jetson/camera.go

```diff
@@ -0,0 +1,80 @@
+package jetsoncamera
+
+import (
+	"bytes"
+	"fmt"
+	"os"
+	"runtime"
+	"syscall"
+)
+
+func GetOSInformation() OSInformation {
+	osInfo := OSInformation{
+		Name:   runtime.GOOS,
+		Arch:   runtime.GOARCH,
+		Kernel: getKernelVersion(),
+		Device: getDeviceName(),
+	}
+	return osInfo
+}
+
+func getKernelVersion() string {
+	var utsname syscall.Utsname
+	if err := syscall.Uname(&utsname); err != nil {
+		return ""
```

Comment:
maybe write unknown?

Reply:
Now returns `"unkown"` if unable to retrieve info.